site/content/contribute/mobile/developer-setup/_index.md

```diff
@@ -17,15 +19,15 @@ A macOS computer is required to build the Mattermost iOS mobile app.
 
 Install the following prerequisite software to develop and build the iOS or Android apps. For macOS, we recommend using [Homebrew](https://brew.sh) as a package manager.
 
+#### Install [NodeJS](https://nodejs.org/en/).
+This includes NPM which is also needed. Currently, any version greater than or equal to 10.12.0 is **required** with npm 6.4.1 or higher. Please note that NodeJS version 11.x is **not** working.
+
```

Comment:
for me Node 12 & 14 are working just fine.

Reply:
@haardikdharma10, after fixing the merge conflicts, I will leave you with the remaining feedback from @enahum!

Reply:
Sure thing @jfrerich, thanks a lot!

Reply:
Hi @enahum, what change do you expect here?

Reply:
@haardikdharma10 I would mention that node 12 and 14 are also working versions

Reply:
Thanks @enahum, Hope we can resolve this conversation now!